runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/EngineConfiguration.java

```diff
@@ -284,6 +294,19 @@ private static int defaultBufferPoolCapacity(
         return ENGINE_BUFFER_SLOT_CAPACITY.get(config) * 64;
     }
 
+    private static URL configURL(Configuration config, String url)
+    {
+        try
+        {
+            return URI.create(url).toURL();
+        }
+        catch (MalformedURLException ex)
+        {
+            LangUtil.rethrowUnchecked(ex);
+            return null;
+        }
```

Comment:
Please use single `return` for these `rethrowUnchecked` blocks.

```
String configURL = null;

try
{
    configURL = URI.create(url).toURL();
}
catch (MalformedURLException ex)
{
    LangUtil.rethrowUnchecked(ex);
}

return configURL;
```

Reply:
Done.